es/index.html

```diff
@@ -86,7 +75,7 @@ <h1>No pidas permiso para preguntar, sólo pregunta.</h1>
   <footer>
     Contenido prácticamente robado de <a href="https://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
     -
-    Traducción de <a href="https://github.com/danirod">@danirod</a>
+    Traducción de <a href="https://github.com/mreduar">@mreduar</a>
```

Comment:
You can add yourself to the credits, but you shouldn't just replace it

Reply:
```suggestion
    Traducción de <a href="https://github.com/danirod">@danirod</a> & <a href="https://github.com/mreduar">@mreduar</a>
```

Reply:
> @danirod do you want to review this since you made the original?

Taking a look